ntbs-service/Services/ValidationService.cs

```diff
@@ -34,7 +34,16 @@ public ValidationService(PageModel pageModel)
 
         public ContentResult GetPropertyValidationResult(object model, string key, object value)
         {
-            SetProperty(model, key, value);
+            try
+            {
+                SetProperty(model, key, value);
+            }
+            catch (Exception)
+            {
+                var propertyDisplayName = model.GetType().GetProperty(key).GetCustomAttribute<DisplayAttribute>()?.Name;
```

Comment:
Should we use the name of the property here as fallback where `DisplayAttribute` is not there?